### PR TITLE
Improve Ukrainian language

### PR DESCRIPTION
**Pull request by vitaliy172**

### CODE REVIEW

**Review by Antti — APPROVED** [review]
LGTM

**Comment by Antti** [comment]
@vitaliy172 I think you also need to run `buildroot/scripts/auto_gen_language_pack.py` to generate the language packs.

**Comment by vitaliy172** [comment]
Yeah but autogenerated translations isn`t that good. I`m native Ukrainian so I can translate a bit better

File: Copy to SD Card root directory to update/Language Packs/language_uk.ini [hunk 1]
```diff
@@ -202,8 +202,8 @@ label_move_speed:Швидкість (X Y Z)
 label_knob_led:Підсвічування енкодера
 label_knob_led_idle:Простій підсвічування енкодера
 label_m0_pause:Пауза по команді M0
-label_send_start_gcode:G-код перед друком
-label_send_end_gcode:G-код після друку
+label_send_start_gcode:Розпочати G-код перед друком
```
Comment (Antti) [inline Copy to SD Card root directory to update/Language Packs/language_uk.ini:205]: I don't think these are correct. You don't "begin" or "end" a G-code.
It toggles sending start/end g-code On or Off.

Comment (vitaliy172) [reply Copy to SD Card root directory to update/Language Packs/language_uk.ini:205]: English version:
`label_send_start_gcode:Start Gcode before print
label_send_end_gcode:End Gcode after print`

Comment (Antti) [reply Copy to SD Card root directory to update/Language Packs/language_uk.ini:205]: I think "Start Gcode" means the gcode, which is exected on the start.
Same with the "End Gcode" - gcode, which is executed on the end.

**Comment by Antti** [comment]
`label_send_start_gcode` and `label_send_end_gcode` are incorrect 😞

**Pull request merged**